include/snitch/snitch_registry.hpp

```diff
@@ -37,6 +37,8 @@ constexpr std::size_t max_unique_tags = SNITCH_MAX_UNIQUE_TAGS;
 constexpr std::size_t max_registered_reporters = SNITCH_MAX_REGISTERED_REPORTERS;
 // Maximum size of a reporter instance, in bytes.
 constexpr std::size_t max_reporter_size_bytes = SNITCH_MAX_REPORTER_SIZE_BYTES;
+// Is snitch disabled?
+constexpr bool is_disabled = !SNITCH_ENABLE;
```

Comment:
If this isn't used, it can be removed. Otherwise, I'd rename it to `is_enabled` to match the macro.